src/http/server/handlers/static_file_handler.cr

```diff
@@ -111,10 +111,7 @@ class HTTP::StaticFileHandler
   end
 
   private def redirect_to(context, url)
-    context.response.status = :found
-
-    url = URI.encode(url.to_s)
-    context.response.headers.add "Location", url
+    context.response.redirect url.to_s
```

Comment:
Maybe redirect could also accept a `Url`?

Reply:
Yes, an overload for `URI` would be an option. In `StaticFileHandler` it's actually a `Path`, though. And that could also be a viable type.

Reply:
So many types...